package.json

```diff
@@ -30,8 +30,16 @@
     "url": "https://github.com/pluralsight/git-switch-electron"
   },
   "dependencies": {
+    "babel-cli": "6.18.0",
+    "babel-core": "6.20.0",
+    "babel-eslint": "7.2.1",
+    "babel-loader": "6.2.10",
+    "babel-plugin-transform-async-to-generator": "6.24.1",
     "babel-plugin-transform-decorators-legacy": "1.3.4",
-    "babel-preset-es2015": "6.22.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.18.0",
+    "babel-plugin-transform-object-rest-spread": "6.20.2",
+    "babel-polyfill": "6.20.0",
+    "babel-preset-es2017": "6.16.0",
```

Comment:
Do all these really need to be prod dependencies?

Reply:
Unfortunately electron-forge needs everything in the compilerrc to be a prod dep. And with babel-polyfill, we are requiring it inline in our entry file. Kinda sucky, but I couldnt find a way around it without forcing us to write vanilla node.

Reply:
:disappointed: